AG-13051 Update tooltip&highlight on click events

olegat opened the pull request:
https://ag-grid.atlassian.net/browse/AG-13051

olegat committed:
AG-13051 Update tooltip&highlight on click events

alantreadway reviewed (approved):
LGTM, one snapshot failure though

olegat commented:
At first glance, the snapshot failure look like a genuine bug regression on an edge case. I'll look into it.

olegat committed:
AG-13051 Update snapshot

The old snapshot was incorrect because the doubleClickAction does not fire a
mouse hover event (which would update the highlight).

We should update the doubleClickAction and clickAction to fire a mouse hover,
but that triggers other snapshot failures, so just update this snapshot for now.

olegat commented on packages/ag-charts-enterprise/src/features/zoom/__image_snapshots__/zoom-test-ts-zoom-listeners-should-not-zoom-out-when-double-clicking-a-node-1-snap.png:
> [!IMPORTANT]
> This new snapshot is correct. The old snapshot does not include the crosshair/highlight because the [`doubleClickAction`](https://github.com/ag-grid/ag-charts/blob/2492a51fdaef4e867620d7d6d0356ac0755030d1/packages/ag-charts-community/src/chart/test/utils.ts#L352) test function does not dispatch a `'mousemove'` event before the first `click` event.

I'll update the `clickAction` and `doubleClickAction` to fire `'mousemove'` events in a separate PR because doing so triggers other unrelated snapshot updates.

alantreadway reviewed (approved):
LGTM
